qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         proxies: Optional[dict] = None,
         verify: Optional[bool] = None,
         channel_strategy: Optional[str] = None,
+        private_endpoint: Optional[bool] = False,
```

Comment:
Shouldn't the default be `None`? So I can choose _not_ to connect to private endpoint even if my saved account says otherwise? 

```suggestion
        private_endpoint: Optional[bool] = None,
```